plugin/src/main/java/io/jenkins/plugins/casc/impl/secrets/VaultSecretSource.java

```diff
@@ -9,24 +9,22 @@
 
 import java.io.FileInputStream;
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
+import java.util.*;
```

Comment:
not sure if there's a coding style here, personally I normally avoid star import so you can see at a glance what is used

Reply:
:+1: